UnityProject/Assets/Scripts/Items/Implants/Organs/BodyPartUISlots.cs

```diff
@@ -24,25 +24,6 @@ public class BodyPartUISlots : Organ, IDynamicItemSlotS
 		[SerializeField] [FormerlySerializedAs("Storage")]
 		private List<BodyPartUISlots.StorageCharacteristics> storage;
 
-		public override void RemovedFromBody(LivingHealthMasterBase livingHealthMasterBase)
```

Comment:
where is this stuff handled now?

Reply:
SetHealthMaster() in BodyPart

Reply:
why Separate it out?

Reply:
more organized that way, plus this is no longer an override since its not longer an organ but a monobehaviour